src/main/java/org/jabref/preferences/JabRefPreferences.java

```diff
@@ -2652,8 +2657,8 @@ public XmpPreferences getXmpPreferences() {
                 (obs, oldValue, newValue) -> putBoolean(USE_XMP_PRIVACY_FILTER, newValue));
         xmpPreferences.getXmpPrivacyFilter().addListener((SetChangeListener<Field>) c ->
                 putStringList(XMP_PRIVACY_FILTERS, xmpPreferences.getXmpPrivacyFilter().stream()
-                                                                 .map(Field::getName)
-                                                                 .collect(Collectors.toList())));
+                        .map(Field::getName)
```

Comment:
All of these reformat changes means that the local IDE in which this change was written does not use the project code style